packages/@sanity/core/src/commands/dataset/copyDatasetCommand.js

```diff
@@ -54,6 +57,30 @@ const progress = (url) => {
   })
 }
 
+const followProgress = (jobId, client, output) => {
+  const spinner = output
+    .spinner({
+      text: `~0% done.`,
+    })
+    .start()
+
+  const listenUrl = client.getUrl(`jobs/${jobId}/listen`)
+
+  debug(`Listening on ${listenUrl}`)
```

Comment:
Tiny Nitpick: 'Listening on' is used by host and not the clients. Alternative is 'Listening to'.

Reply:
Fixed in 77c78deda 🙂